src/api/client/Claims.ts

```diff
@@ -23,16 +23,24 @@ import {
 import { Ensured } from '~/types/utils';
 import { DEFAULT_GQL_PAGE_SIZE } from '~/utils/constants';
 import {
+  bigNumberToU32,
+  bytesToString,
   identityIdToString,
   meshClaimToClaim,
   momentToDate,
   scopeToMiddlewareScope,
   signerToString,
   stringToIdentityId,
   toIdentityWithClaimsArray,
+  u32ToBigNumber,
 } from '~/utils/conversion';
 import { calculateNextKey, createProcedureMethod, getDid, removePadding } from '~/utils/internal';
 
+type CustomClaimTypeToHuman = {
+  name: string;
+  id: string;
+};
```

Comment:
```suggestion
export type CustomClaimType = {
  name: string;
  id: BigNumber;
};
```
Why not just have the name to `CustomClaimType`? Also since this is used as part of return type of a method we need to export it so that docs don't complain (so maybe move it to types file)